.github/workflows/test.yml

```diff
@@ -8,7 +8,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        SETUP: ['/cvmfs/sw.hsf.org/key4hep/setup.sh', 'init_lcg_97a_FCC_4.sh']
+        SETUP: ['/cvmfs/sw.hsf.org/key4hep/setup.sh', 'init_lcg_99.sh']
```

Comment:
The `init_lcg_99.sh` seems to be missing, resulting in a failing CI.

Reply:
ah, yes. The problem with the lcg releases is that even in the latest one the delphes version is too old for the covariance code in the converter. I'll just remove it.